### DISCUSSION TITLE
Selling individual tracks

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#509] [Ideas]
Artists could sell individual tracks on albums. 

I think for cost benefit, there'd have to be a minimum price for tracks (considering Stripe takes a 30c minimum cut). 

Steps: 
* [x] Makes tracks visible as their own page
* [x] Verify and properly store metadata on tracks
* [x] Use metadata to display extra information about tracks
* [x] Build out purchaseTrack workflow

**Comment by thevril** [comment]
> I think for cost benefit, there'd have to be a minimum price for tracks (considering Stripe takes a 30c minimum cut).

or free download :)

**Comment by simonv3** [comment]
This is now public with #1148
